tendrl/node_agent/manager/__init__.py

```diff
@@ -116,6 +123,9 @@ def shutdown(signum, frame):
             NS.tendrl_context.integration_id:
             NS.ceph_integrations_sync_thread.stop()
 
+        if "tendrl/monitor" in NS.node_context.tags:
```

Comment:
You'll need to check

NS.tendrl_context.sds_name == "gluster"

before starting the gluster sync thread